packages/warehouses/src/warehouseClients/PostgresWarehouseClient.ts

```diff
@@ -170,12 +170,23 @@ export class PostgresClient<
             rows: Record<string, any>[];
         }>((resolve, reject) => {
             pool = new pg.Pool(this.config);
+
             pool.connect((err, client, done) => {
+                if (!client) {
```

Comment:
when is the client undefined?

Reply:
I upgrade the library and its types. It seems connect can return undentified clients, so I'm capturing this as an error